### PR TITLE
NoticeBoard is craftable now

### PR DESCRIPTION
**Pull request by TokenStyle**
## About the PR

Made Notice Board craftable. It's made of 3 plain wood planks. Could be found in crafting panel > furniture

## Why / Balance

#24901 Notice board it's just wood board, so it's obvious it could be constructed.

## Media

snapshot 2024-04-09

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Notice board can be crafted

### CODE REVIEW

**Comment by Killerqu00** [comment]
5 wood would be better imho

**Comment by Terraspark4941** [comment]
> noticeBoard

needs a space and proper capitalization, not just camelCase

> Notice board

**Comment by TokenStyle** [comment]
Fixed naming and desc

**Comment by TokenStyle** [comment]
> 5 wood would be better imho

Yes it could, though bookshelf costs 5 wood planks, and dresser costs 3 wood, I honesly don't think that paper storage would cost the same amount of material as bookshelf

**Comment by beck-thompson** [comment]
I think 3 wood is probably fine! I don't see why 5 vs 3 would make any difference realistically.

File: Resources/Prototypes/Recipes/Construction/furniture.yml [hunk 1]
```diff
@@ -884,3 +884,21 @@
...
+  category: construction-category-furniture
+  icon:
+    sprite: Structures/Wallmounts/noticeboard.rsi
+    state: noticeboard
+  objectType: Structure
+  placementMode: SnapgridCenter
+  canRotate: true
+  canBuildInImpassable: false
+  conditions:
+    - !type:TileNotBlocked
```
Comment (metalgearsloth) [inline Resources/Prototypes/Recipes/Construction/furniture.yml:904]:
```suggestion
    - !type:TileNotBlocked

```

**Pull request merged**